src/migrators/HimalayaConnext.sol

```diff
@@ -68,13 +79,17 @@ contract HimalayaConnext is IXReceiver, IHimalayaConnext {
     migration.assetDest.safeApprove(migration.himalaya, migration.amount);
 
     //Handle inbound
+    //TODO this call should be wrapped in a try-catch.
```

Comment:
Added this todo. We have to wrap the call to the HimalayaCompound in a try catch. 
If we hard-revert funds get "trapped" on Connext contract.

Reply:
Once we add the try-catch this affects also some of the tests you implemented.